### PR TITLE
Bug 1200392 - Remove wait_for methods in base.py, part 2

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=df7922907e05bae5a540c2ed039aa30b825e8d79

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3322d8d6acf308ff45d439d6b104e42e2f29ee19

File: tests/python/gaia-ui-tests/gaiatest/gaia_test.py [hunk 1]
```diff
@@ -1096,32 +1096,6 @@ def set_default_settings(self):
...
-    def wait_for_element_not_displayed(self, by, locator, timeout=None):
-        self.marionette.set_search_timeout(0)
-        try:
-            Wait(self.marionette, timeout, ignored_exceptions=StaleElementException).until(
-                lambda m: not m.find_element(by, locator).is_displayed())
-        except NoSuchElementException:
-            pass
-        self.marionette.set_search_timeout(self.marionette.timeout or 10000)
-
     def wait_for_condition(self, method, timeout=None, message=None):
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/gaia_test.py]: Do we need to keep this function?

File: tests/python/gaia-ui-tests/gaiatest/apps/base.py [hunk 2]
```diff
@@ -24,9 +24,6 @@ def __init__(self, marionette):
     def launch(self, launch_timeout=None):
         self.app = self.apps.launch(self.name, self.manifest_url, self.entry_point, launch_timeout=launch_timeout)
 
-    def wait_for_condition(self, method, timeout=None, message=None):
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/base.py]: I think 2 more calls of this function are missing, in:
* `test_dialer.py`
* `test_dialer_find_contact.py`

File: tests/python/gaia-ui-tests/gaiatest/gaia_test.py (hunk 1 above)
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/gaia_test.py]: Eh, you commented on it in the bug

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/gaia_test.py]: Yes, we can get rid of it, but there are lots of uses of it, so I'd rather do it in a separate bug then.

File: tests/python/gaia-ui-tests/gaiatest/apps/base.py (hunk 2 above)
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/base.py]: Good catch, thanks!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40ff937067af33a1146fcf9261f6abbbf56d80f0

**Pull request merged**